src/murmur3_x64_128.rs

```diff
@@ -119,6 +120,124 @@ pub fn murmur3_x64_128<T: Read>(source: &mut T, seed: u32) -> Result<u128> {
     }
 }
 
+/// Use the x64 variant of the 128 bit murmur3 to hash byte slice without copying the buffer.
+///
+/// # Example
+/// ```
+/// use murmur3::murmur3_x64_128_of_slice;
+/// let hash_result = murmur3_x64_128_of_slice(b"hello world", 0);
+/// ```
+pub fn murmur3_x64_128_of_slice(source: &[u8], seed: u32) -> Result<u128> {
```

Comment:
This is returning a Result, but I believe the operation is infallable.  Is that correct?  Converting this to result u128 would probably be my only change

Reply:
Updated.